server/balancer.go

```diff
@@ -124,29 +124,35 @@ func (cb *capacityBalancer) selectToStore(stores []*storeInfo, excluded map[uint
 	return resultStore
 }
 
-func (cb *capacityBalancer) selectBalanceRegion(cluster *clusterInfo, stores []*storeInfo) (*metapb.Region, *metapb.Peer) {
+func (cb *capacityBalancer) selectBalanceRegion(cluster *clusterInfo, stores []*storeInfo) (*metapb.Region, *metapb.Peer, bool) {
```

Comment:
If the bool is true, it means we select a leader peer. Otherwise, we select a follower peer.